app/assets/stylesheets/rentals/rental_schedule.css

```diff
@@ -17,7 +17,8 @@ table, th, td {
 }
 
 .simple-calendar table td {
-  height: 100px !important;
+  height: 123px !important;
+  width : 163px !important;
```

Comment:
Also, may be getting rid of simple calendar in my next pull request in favor of a simpler form for scheduling golf carts.